code/plotting/extended_data_fig5.do

```diff
@@ -128,13 +128,11 @@ graph export results/figures/appendix/fixed_lag/fig5_FL.pdf, replace
 // output source data for ED fig 5
 // Will pass "nosave" as argument during tests so that we don't overwrite
 if "`0'" == "" {
-    export excel adm0 policy beta lower upper using "results/source_data/ExtendedDataFigure5_lags.xlsx", sheet("panel_a") firstrow(var) sheetreplace
-
+    preserve
     import delim "results/source_data/indiv/ExtendedDataFigure5_b.csv", clear
-    export excel using "results/source_data/ExtendedDataFigure5_lags.xlsx", sheet("panel_b") firstrow(var) sheetreplace
-
-    import delim "results/source_data/indiv/ExtendedDataFigure5_CHN_event_study.csv", clear
-    export excel using "results/source_data/ExtendedDataFigure5_lags.xlsx", sheet("panel_c") firstrow(var) sheetreplace
+    export excel using "results/source_data/ExtendedDataFigure5_lags.xlsx", sheet("panel_a") firstrow(var) sheetreplace
```

Comment:
change sheetreplace to replace in line 133

Reply:
fixed!